core/pom.xml

```diff
@@ -150,6 +150,11 @@
             <groupId>org.apache.commons</groupId>
             <artifactId>commons-lang3</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.python</groupId>
+            <artifactId>jython</artifactId>
+            <version>2.7.0</version>
```

Comment:
Agree that jython is better left as an optional addon - not worth the weight it brings in. Plus we already bundle groovy and javascript is provided by the JRE so there are alternatives to choose from.